personal_assistant/pa_note/forms.py

```diff
@@ -0,0 +1,7 @@
+from django import forms
+from .models import Note
+
+class NoteForm(forms.ModelForm):
+    class Meta:
+        model = Note
+        fields = ['name', 'description', 'done']
```

Comment:
Немає сенсу створювати нотатку, яку вже виконано тож відповідне поле варто тут сховати і створити окремий маршрут, який відмічатиме нотатку як виконану:
```suggestion
        fields = ['name', 'description']
```